### DISCUSSION TITLE
interface for clickhouse

### DISCUSSION THREAD

**Discussion by yongjinhou** [ClickHouse/ClickHouse#45813] [Q&A] [answered]
Whether the interface provided by clickhouse supports https protocol. How does https work if it is supported.

**Comment by den-crane** [comment]
Yes. It does.

https://docs.altinity.com/operationsguide/security/clickhouse-hardening-guide/network-hardening/#enable-tls

**Reply by yongjinhou** [reply]
Thank you very much.

**Reply by yongjinhou** [reply]
I see that the https interface description is not updated in the wiki. [https://clickhouse.com/docs/zh/interfaces/](url)

**Reply by mdonkers** [reply, marked as answer]
I'm not sure where the link was supposed to point to, but now getting "page not found" for the docs/zh/interfaces link.

Our ClickHouse docs do explain how to configure SSL/TLS though: https://clickhouse.com/docs/en/guides/sre/configuring-ssl
